iot2000.coffee

```diff
@@ -0,0 +1,35 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+module.exports =
+	version: 1
+	slug: 'IOT2000'
+	aliases: [ 'IOT2020', 'IOT2040' ]
+	name: 'Siemens IOT2000'
+	arch: 'i386'
+	state: 'preview'
+
+	instructions: commonImg.instructions
+
+	gettingStartedLink:
+		windows: 'https://docs.resin.io/iot2000/nodejs/getting-started/#adding-your-first-device'
+		osx: 'https://docs.resin.io/iot2000/nodejs/getting-started/#adding-your-first-device' 
+		linux: 'https://docs.resin.io/iot2000/nodejs/getting-started/#adding-your-first-device'
```

Comment:
Did you also ping Zach about these?

Reply:
@telphan I need info for two sections to create a getting started guide: What You Will Need, and Setting Up Your Device. Feel free to copy the sections from the devices that are closest to this new device and make modifications as necessary

Reply:
@telphan did you provide to @zwalchuk what he has asked for?

Reply:
@floion provided internally.